### PR TITLE
Add 2.3

### PR DESCRIPTION
**Pull request by chneau**
Based on PR https://github.com/disney/meteor-base/pull/69 

Changelog: https://github.com/meteor/meteor/blob/devel/History.md

Now, by default `meteor create` uses the react template. https://react-tutorial.meteor.com/simple-todos/01-creating-app.html#1-2-Create-Meteor-Project 
while the Blaze one now needs to be specified with `--blaze`. https://blaze-tutorial.meteor.com/simple-todos/01-creating-app.html#1-2-Create-Meteor-Project 

`example/app` has been deleted and recreated with this command `meteor create --release 2.3 app`.

Here is the new version of babel `"@babel/runtime": "^7.14.0"` from `"@babel/runtime": "^7.12.5",` for Meteor 2.2.

There is a bump in the node version used:
`Node.js update to 14.17.1 from 12.22.1`

### CODE REVIEW

**Comment by chneau** [comment]
Latest commit makes tests pass on my side https://github.com/chneau/meteor-base/actions/runs/1001289843 

```bash
	node_version='12.22.1'
	# Versions < 1.9 need Node 8.17.0
	if [[ "${version}" == 1.6* ]] || [[ "${version}" == 1.7* ]] || [[ "${version}" == 1.8* ]]; then
		node_version='8.17.0'
	fi

	# Versions 2.3 need Node 14.17.1
	if [[ "${version}" == 2.3* ]]; then
		node_version='14.17.1'
	fi
```

**Comment by diavrank** [comment]
Maybe it is better idea to add first [Meteor 2.2.1](https://github.com/meteor/meteor/blob/devel/History.md#v221-2021-06-02), since there was a new node version (**12.22.2**). I've been waiting for the new [alpine docker](https://hub.docker.com/_/node) image to do PR.

**Comment by chneau** [comment]
Hi @diavrank , I'll be happy to do the PR for 2.2.1, if it follows the same logic as this PR that should be easy enough. If you give me the green light I could do the PR in a few hours after getting back from the gym.

**Comment by diavrank** [comment]
Yes, no problem for me :) .

**Comment by diavrank** [comment]
[Meteor 2.3.1](https://github.com/meteor/meteor/blob/devel/History.md#v231-2021-07-08) has been released few days ago. @chneau can you help me with a PR? This is to leverage the moment when the PRs start to be merged.

**Comment by chneau** [comment]
@diavrank Hello, no problem I'll be happy to create the PR for Meteor 2.3.1 following the same process :)

File: test.sh [hunk 1]
```diff
@@ -62,6 +62,11 @@ for version in "${meteor_versions[@]}"; do
 		node_version='8.17.0'
 	fi
 
+	# Versions 2.3 need Node 14.17.1
+	if [[ "${version}" == 2.3* ]]; then
+		node_version='14.17.1'
+	fi
```
Comment (GeoffreyBooth) [inline test.sh:68]: Untested, but I think we want something like this. The previous code was dealing with only two potential Node versions, but now we have more (and upcoming Meteor versions add even more):

```shell
	# Versions 1.8.x and below need Node 8.17.0
	if [[ "${version}" == 1.6* ]] || [[ "${version}" == 1.7* ]] || [[ "${version}" == 1.8* ]]; then
		node_version='8.17.0'

	# Versions 1.9 through 2.2.x need Node 12.22.1
	elif [[ "${version}" == 1.9* ]] || [[ "${version}" == 2.0* ]] || [[ "${version}" == 2.1* ]]; then
		node_version='12.22.1'

	# Versions >= 2.3 need Node 14.17.1
	else
		node_version='14.17.1'
	fi
```

Comment (chneau) [reply test.sh:70]: I modified the code like in the code you provided. Tests are passing properly https://github.com/chneau/meteor-base/actions/runs/1027745165

**Comment by GeoffreyBooth** [comment]
Oops, I merged these out of order. Will fix in a bit.

**Pull request merged**